truncate/table_schema.go

```diff
@@ -96,18 +111,6 @@ func fetchTableSchemas(ctx context.Context, client *spanner.Client, targetTables
 			return err
 		}
 
-		if !truncateAll {
-			if len(excludes) != 0 {
-				if _, ok := excludes[tableName]; ok {
-					return nil
-				}
-			} else {
-				if _, ok := targets[tableName]; !ok {
-					return nil
-				}
-			}
-		}
-
```

Comment:
To avoid excessive nesting from adding the new if statement, I have opted to use a switch statement instead.